packages/components/timed-text-editor/index.js

```diff
@@ -509,6 +509,7 @@ class TimedTextEditor extends React.Component {
           blockRendererFn={ this.renderBlockWithTimecodes }
           handleKeyCommand={ command => this.handleKeyCommand(command) }
           keyBindingFn={ e => this.customKeyBindingFn(e) }
+          spellCheck={ this.props.spellCheck }
```

Comment:
```suggestion
          spellCheck={ this.props.spellCheck?  this.props.spellCheck : false  }
```

What do you think about having a default to false if the prop is not specified to making it an optional parameter to the component?

Reply:
This parameter is already optional. `this.props.spellCheck` will evaluate to `true`, `false`, or `undefined`. All 3 are valid values. Additionally, the relevant prop type is not suffixed with `isRequired`.